### DISCUSSION TITLE
FPS Game Sample

### DISCUSSION THREAD

**Discussion by VinnyVicious** [o3de/rfcs#32] [Ideas]
**Is your feature request related to a problem? Please describe**.
Sample projects enable users to learn about the engine iteratively, slowly making progress by analyzing different parts of the project and making sense of the overall architecture. They are also great ways to document best practices and recommend approaches to common problems. Sample projects are key in to making O3DE widely accessible.

**Describe the solution you'd like**
A fully-featured FPS game sample, with melee, raycast and projectile-based weapon systems, character controller and simple AI. 

**Describe alternatives you've considered**
Port existing Lumberyard game samples, or Legacy GameSDK.

**Comment by kberg0** [comment]
The MultiplayerSample repo will eventually be what you are looking for. We're still working on flushing out the core functionality.
